src/pages/ExamsPage/components/ExternalReviewDashboard.jsx

```diff
@@ -6,9 +6,23 @@ import { getExamsBaseUrl } from '../data/api';
 const getLaunchUrlByExamId = (id) => `${getExamsBaseUrl()}/lti/exam/${id}/instructor_tool`;
 
 const ExternalReviewDashboard = ({ exam }) => (
+  // By default, launch instructor tool in a new tab.
+  // Keep code path to embed as iframe for now, so that either option can eventually be supported.
   <div data-testid="review_dash">
-    {exam && <iframe title="lti_tool" src={getLaunchUrlByExamId(exam.id)} width="100%" height="1100" style={{ border: 'none' }} />}
+    <div>
+      {!process.env.LTI_INSTRUCTOR_EMBED && exam && window.open(getLaunchUrlByExamId(exam.id), '_blank')}
```

Comment:
defaults to opening in new tab (based on env settings) and keeps logic below to embed as iframe.